src/cc/bpf_module.h

```diff
@@ -117,9 +117,11 @@ class BPFModule {
   char * license() const;
   unsigned kern_version() const;
   TableStorage &table_storage() { return *ts_; }
+  void disable_rw_engine() { rw_engine_enabled_ = false; }
```

Comment:
This is a bit tricky. For Python (even if we expose it), the loading is already done at construction time and if user call this `disable_rw_engine` later there would be no effect / saving on memory. For C++, it must be called before `BPF::init()` otherwise same thing. And this gives people the impression that it could be turned on and off on-demand, rather than a one-off switch at load time. How about we just keep it at the constructor?

Reply:
Adding to constructor making sense. Will add C++ API as well.